src/client/app/components/MeterDropDownComponent.jsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+
+export default class MeterDropDownComponent extends React.Component {
```

Comment:
We have similar-ish components that accomplish this. See MultiSelect component or SingleSelectComponent.